packages/autolink/src/Mention.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from './Link';
+import { MentionProps } from './types';
+
+export function Mention({ children, mention, mentionUrl, ...props }: MentionProps) {
+	if (!mentionUrl) {
+		return null;
```

Comment:
This should return `mention`, otherwise when parsed within a string it would replace incorrectly.

For example, "hello @name" would become "hello " when no URL is present.